### PR TITLE
OKTA-149608: Fully documents Logs API query parameters.

### PR DESCRIPTION
**Pull request by bobtiernay-okta**
## Description:

Fully documents Logs API query parameters in preparation for EA.

### Resolves:

* [OKTA-149608](https://oktainc.atlassian.net/browse/OKTA-149608)

### CODE REVIEW

File: _source/_docs/api/resources/system_log.md [hunk 1]
```diff
@@ -26,14 +26,15 @@ The System Log API has one endpoint:
 
 [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9cfb0dd661a5432a77c6){:target="_blank"}
 
-See [Examples](#examples) for ways you can use the System Log API. For common use cases see [Useful System Log Queries](https://support.okta.com/help/Documentation/Knowledge_Article/Useful-System-Log-Queries).
+This collection resource is backed by a rich [LogEvent object](#logevent-object) model and associated [event types](#event-types).
```
Comment (omgitstom) [inline _source/_docs/api/resources/system_log.md]: > The `logs` collection resource contains [LogEvent object](#logevent-object) model and associated [event types](#event-types) for your Okta Organization.

I think the above statement gets to the heart of what you are attempting to convey

Comment (bobtiernay-okta) [reply _source/_docs/api/resources/system_log.md]: Will update, thanks!

Comment (bobtiernay-okta) [reply _source/_docs/api/resources/system_log.md]: Although is the grammar correct here: "contains LogEvent object model "?

Seems like it needs an article.

**Comment by bobtiernay-okta** [comment]
@omgitstom I'm wondering if we should tease out your new comments into a new PR? I think more discussion would need to go into these items, associated by another JIRA.

**Comment by mystiberry-okta** [comment]
@omgitstom & @bobtiernay-okta , okay to merge with the other JIRA I created (see slack)?

**Comment by bobtiernay-okta** [comment]
I'm good with that. @omgitstom what say you?

File: _source/_docs/api/resources/system_log.md [hunk 2]
```diff
@@ -189,11 +77,11 @@ Each Log object describes a single action performed by a set of actors for a set
     "externalSessionId": "1013FfF-DKQSvCI4RVXChzX-w"
   }
 }
-~~~
+```
 
-### Log Object annotated Example
+### LogEvent Object Annotated Example
```
Comment (bdemers) [inline _source/_docs/api/resources/system_log.md]: Can we get rid of this section? None of the other endpoints are structure like this. And this  is a read only endpoint, so nothing is `Required` or `Optional` from the User's point of view. (`nullable` maybe, but still I don't think that is needed either)

**Comment by bdemers** [comment]
`ipChain` is still listed under `Client` which is wrong.  

it is under `request`
```json
[{
...
"request": {
            "ipChain": [
                {
                    "ip": "66.222.111.88",
                    "geographicalContext": {
                        "city": "Concord",
                        "state": "New Hampshire",
                        "country": "United States",
                        "postalCode": "03303",
                        "geolocation": {
                            "lat": 43.3091,
                            "lon": -71.6861
                        }
                    },
                    "version": "V4",
                    "source": null
                }
...
}]
```

**Comment by bdemers** [comment]
`client.userAgent` is a complex object, not a string:

``` json
"userAgent": {
                "rawUserAgent": "okta-sdk-java/0.9.0-SNAPSHOT jetty/9.2.22.v20170606 java/1.8.0_121 Mac OS X/10.13.1",
                "os": "Mac OS X",
                "browser": "UNKNOWN"
            }
```

**Comment by bdemers** [comment]
image
empty lines in table

**Comment by bobtiernay-okta** [comment]
Confused by some of these comments.  Although appreciated and great feedback, they seem to be out of scope with respect to OKTA-149608. Can we circle back and add them to the ticket @mystiberry-okta created?

**Comment by mystiberry-okta** [comment]
@bobtiernay-okta JIRA OKTA-152123 is for all outstanding comments on this PR.

**Comment by bobtiernay-okta** [comment]
Removed the missing table rows and squashed. Will address the other feedback in aforementioned ticket.

File: _source/_docs/api/resources/system_log.md [hunk 3]
```diff
@@ -277,7 +277,7 @@ Describes the client performing the action
 
 ### GeographicalContext Object
 
-Describes the location of the target that the action was performed on
+Describes the location of the target that the action was performed on.
```
[the remark this answers is not in the document]
Comment (ericzhou-okta) [reply _source/_docs/api/resources/system_log.md]: That would make more sense. I see little reason why anyone would care about the target geolocation, when targets are often policies and rules, and we can't access a person's current location.

**Pull request merged**